astilectron.go

```diff
@@ -182,13 +193,14 @@ func (a *Astilectron) provision() error {
 	return a.provisioner.Provision(ctx, a.options.AppName, runtime.GOOS, runtime.GOARCH, *a.paths)
 }
 
-// listenTCP listens to the first TCP connection coming its way (this should be Astilectron)
+// listenTCP creates a TCP server for astilectron to connect to
+// and listens to the first TCP connection coming its way (this should be Astilectron).
 func (a *Astilectron) listenTCP() (err error) {
 	// Log
 	astilog.Debug("Listening...")
 
 	// Listen
-	if a.listener, err = net.Listen("tcp", "127.0.0.1:"); err != nil {
+	if a.listener, err = net.Listen("tcp", "127.0.0.1:"+a.options.TCPPort); err != nil {
```

Comment:
Since the port will be `*int` I'd rather have the `addr` be computed just above. It should default to `127.0.0.1:` and have a port only if the value `!= nil`

Reply:
Done.